### PR TITLE
add infoblox dns provider support

### PR DESCRIPTION
**Pull request by aiwantaozi**
Related Issue:  rancher/rancher#8546
Infoblox related DNS code: https://github.com/rancher/external-dns/pull/90
Changes: add Infoblox external DNS support

@yasker  could you help review this, please? thanks

### CODE REVIEW

**Comment by yasker** [comment]
Note: Need to wait for the image to release.

**Comment by aiwantaozi** [comment]
@yasker just update the image name, please help review the PR for infoblox external dns provider.

File: infra-templates/infoblox/0/rancher-compose.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# notemplating
+.catalog:
+  name: "Infoblox DNS"
+  version: "v0.1.0"
+  description: "Rancher External DNS service powered by Infoblox"
+  minimum_rancher_version: v1.2.0-pre4-rc1
```
Comment (deniseschannon) [inline infra-templates/infoblox/0/rancher-compose.yml]: Is this really the right min version to use?

Comment (aiwantaozi) [reply infra-templates/infoblox/0/rancher-compose.yml]: @deniseschannon Hi Denise, this min version just the same as other DNS provider, we tested the infoblox DNS provider in rancher 1.4 and 1.6, how about we change to 1.4?

Comment (deniseschannon) [reply infra-templates/infoblox/0/rancher-compose.yml]: Can you please update it to `v1.6.0`?

Comment (aiwantaozi) [reply infra-templates/infoblox/0/rancher-compose.yml]: Just updated to v1.6.0, thanks.

**Pull request merged**